proto/attestation/reference_value.proto

```diff
@@ -94,21 +95,34 @@ message Regex {
   string value = 1;
 }
 
+message StringLiteral {
+  string value = 1;
+}
+
+// Deprecated. Use TextReferenceValue instead.
```

Comment:
I think you can mark the message itself as deprecated with an annotation.

Reply:
Looks like to deprecate an entire message an option needs to be used